compose/bin/install-datasolutions

```diff
@@ -24,36 +30,40 @@ if [[ "$SAAS_ENVIRONMENT" == "qa" ]]; then
-    sed -i '' -e 's/commerce.adobe.io/commerce-int.adobe.io/g' extensions/catalog-sync-admin/CatalogSyncAdmin/etc/csp_whitelist.xml
-    sed -i '' -e 's/commerce.adobe.net/catalog-sync-ui-qa.magento-datasolutions.com/g' extensions/catalog-sync-admin/CatalogSyncAdmin/etc/csp_whitelist.xml
+
+    if [[ "$INSTALL_PREX" ]]; then
+        sed -i '' -e 's/api.magento.com/qa-api.magedevteam.com/g' extensions/product-recommendations/ProductRecommendationsLayout/etc/csp_whitelist.xml
+        sed -i '' -e 's/commerce.adobe.io/commerce-int.adobe.io/g' extensions/product-recommendations/ProductRecommendationsLayout/etc/csp_whitelist.xml
+        sed -i '' -e 's/magento-recs-sdk.adobe.net/sdk.magento-datasolutions.com/g' extensions/product-recommendations/ProductRecommendationsLayout/etc/csp_whitelist.xml
+        sed -i '' -e 's/api.magento.com/qa-api.magedevteam.com/g' extensions/product-recommendations/PageBuilderProductRecommendations/etc/csp_whitelist.xml
+        sed -i '' -e 's/commerce.adobe.io/commerce-int.adobe.io/g' extensions/product-recommendations/PageBuilderProductRecommendations/etc/csp_whitelist.xml
+        sed -i '' -e 's/magento-recs-sdk.adobe.net/sdk.magento-datasolutions.com/g' extensions/product-recommendations/PageBuilderProductRecommendations/etc/csp_whitelist.xml
+        sed -i '' -e 's/api.magento.com/qa-api.magedevteam.com/g' extensions/product-recommendations-admin/ProductRecommendationsAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/commerce.adobe.io/commerce-int.adobe.io/g' extensions/product-recommendations-admin/ProductRecommendationsAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/commerce.adobe.net/admin-ui-qa.magento-datasolutions.com/g' extensions/product-recommendations-admin/ProductRecommendationsAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/api.magento.com/qa-api.magedevteam.com/g' extensions/product-recommendations-admin/ProductRecommendationsAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/commerce.adobe.io/commerce-int.adobe.io/g' extensions/product-recommendations-admin/ProductRecommendationsAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/api.magento.com/qa-api.magedevteam.com/g' extensions/product-recommendations-admin/ProductRecommendationsSyncAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/commerce.adobe.io/commerce-int.adobe.io/g' extensions/product-recommendations-admin/ProductRecommendationsSyncAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/api.magento.com/qa-api.magedevteam.com/g' extensions/catalog-sync-admin/CatalogSyncAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/commerce.adobe.io/commerce-int.adobe.io/g' extensions/catalog-sync-admin/CatalogSyncAdmin/etc/csp_whitelist.xml
+        sed -i '' -e 's/commerce.adobe.net/catalog-sync-ui-qa.magento-datasolutions.com/g' extensions/catalog-sync-admin/CatalogSyncAdmin/etc/csp_whitelist.xml
+    fi
 fi
 
 sed -i '' -e 's/#- .\/extensions/- .\/extensions/g' docker-compose.dev.yml
```

Comment:
SInce the p-rex and search installs are behind a control flow and this line uncomments all the volume mappings in the docker-compose file, it will break the startup of the container if both p-rex and search aren't installed together.

Reply:
Fixed